### ISSUE TITLE
[Infrastructure][LocalNet] Dynamically Scale Local and/or Remote Environment

### ISSUE THREAD

**Issue by jessicadaugherty** [pokt-network/pocket-operator#8]
## Objective

Use Kubernetes Operator on localnet to send a tx. Blocked by #125 and #7 (Operator).

## Origin Document

[Infra Team Kubernetes Discovery](https://www.notion.so/pocketnetwork/Kubernetes-e1102c252ac54caebac87813e728e320)

## Goals

- [x] Provide a scalable localnet or remote environment K8O deployment structure to send E2E tx.

## Deliverable

- [ ] Implement dynamic scaling for clusters
- [ ] Reduce and add nodes as needed  
- [ ] Use existing config & genesis file generators to scale the local environment

## Non-goals / Non-deliverables

- Scaling or sending txs on devnet
- Localnet using peer discovery or churn

## General issue deliverables

- [x] Update the appropriate CHANGELOG
- [x] Update any relevant READMEs (local and/or global)
- [x] Update any relevant global documentation & references
- [x] If applicable, update the source code tree explanation
- [x] If applicable, add or update a state, sequence or flowchart diagram using [mermaid](https://mermaid-js.github.io/mermaid/)

## [Optional] Testing Methodology

- **Task specific tests**: `make ...`
- **All tests**: `make test_all`
- **LocalNet**: verify a `LocalNet` is still functioning correctly by following the instructions at [docs/development/README.md](https://github.com/pokt-network/pocket/tree/main/docs/development)

---

**Creator**: @jessicadaugherty
**Co-Owners**: @okdas

**Comment by jessicadaugherty** [comment]
Redundant. Already scales.
